.gitlab-ci.yml

```diff
@@ -1271,6 +1272,26 @@ sign-multiarch-manifest:
   before_script:
     - mv tags_to_sign_${MANIFEST} tags_to_sign
 
+xray-scan-docker:
+  only:
+    - main
+  except:
+    - schedules
+  extends: .container-scan
+  stage: xray-scan
+  parallel:
+    matrix:
+      # - CHILD: ["amd64", "arm64", "ppc64le", "2019", "2022"]
```

Comment:
nit: I think the comment on the actual like is enough

Reply:
> nit: I think the comment on the line below is enough

Just wanted to explicitly show all child tags that we publish that we should scan whenever prodsec's template support them.